### PR TITLE
update most remaining references to ZQL

### PR DESCRIPTION
**Pull request by nwt**
The only remaining references are in CHANGELOG.md and zql/docs/README.md
and are left unchanged deliberately.

### CODE REVIEW

File: docs/formats/zng.md [hunk 1]
```diff
@@ -676,7 +676,7 @@ that has not been previously defined according to the DFS order.
 
 ## Appendix B. Recommended Type Coercion Rules
 
-> TBD: it might be better to put this in the ZQL docs
+> TBD: it might be better to put this in the Zed docs
```
Comment (philrz) [inline docs/formats/zng.md:679]:
```suggestion
> TBD: it might be better to put this in the Zed language docs
```

File: expr/function/math.go [hunk 2]
```diff
@@ -212,7 +212,7 @@ func (s *sqrt) Call(args []zng.Value) (zng.Value, error) {
 	if math.IsNaN(x) {
 		// For now we can't represent non-numeric values in a float64,
 		// we will revisit this but it has implications for file
-		// formats, zql, etc.
+		// formats, Zed, etc.
```
Comment (philrz) [inline expr/function/math.go:215]:
```suggestion
		// formats, the Zed language, etc.
```

File: cmd/zed/query/command.go [hunk 3]
```diff
@@ -96,7 +96,7 @@ func New(parent charm.Command, f *flag.FlagSet) (charm.Command, error) {
 	c.procFlags.SetFlags(f)
 	f.BoolVar(&c.verbose, "v", false, "show verbose details")
 	f.BoolVar(&c.stats, "S", false, "display search stats on stderr")
-	f.BoolVar(&c.quiet, "q", false, "don't display zql warnings")
+	f.BoolVar(&c.quiet, "q", false, "don't display Zed warnings")
```
Comment (philrz) [inline cmd/zed/query/command.go:99]: Maybe just drop the `Zed` entirely?

Comment (nwt) [reply cmd/zed/query/command.go:99]:
```suggestion
	f.BoolVar(&c.quiet, "q", false, "don't display warnings")
```

**Review by mattnibs — APPROVED** [review]

**Pull request merged**